src/Microsoft.AspNet.SignalR.SqlServer/SqlMessageBus.cs

```diff
@@ -134,22 +134,23 @@ private void Initialize(object state)
             }
         }
 
-        private void StartReceiving(int streamIndex)
+        private async void StartReceiving(int streamIndex)
```

Comment:
It is called in a sync method,  it will get compile error if use async Task here.